lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -170,6 +170,12 @@ def self.included(includer)
               Chef::Config[:knife][:bootstrap_vault_item][vault].push(item)
               Chef::Config[:knife][:bootstrap_vault_item]
             }
+
+          option :tags,
+            :long => "--tags TAGS",
+            :description => "Comma separated list of tags to apply to the node",
+            :proc => lambda { |o| o.split(/[\s,]+/) },
+            :default => []
```

Comment:
It does feel like this could be "dried up" Seems like we basically have to inherit via copy/paste when new properties are added to https://github.com/chef/chef/blob/master/lib/chef/knife/bootstrap.rb

would be nice if we could add some DSL like:
```
include_option :runlist
include_option :tags
```

That would also mean not having to do the crazy versioning filters in the tests too